src/ncp/ncp_base.cpp

```diff
@@ -1416,7 +1436,7 @@ ThreadError NcpBase::SendPropertyUpdate(uint8_t header, uint8_t command, spinel_
 
     va_start(args, pack_format);
     SuccessOrExit(errorCode = OutboundFrameBegin());
-    SuccessOrExit(errorCode = OutboundFrameFeedPacked("Cii", header, command, key));
+    SuccessOrExit(errorCode = OutboundFrameFeedPacked(SPINEL_DATATYPE_COMMAND_PROP_S, header, command, key));
```

Comment:
In the spec, I use `PROP_ID`. I agree it would be most consistent to use something like `prop_id` instead of `key`, but I don't really know how much more readable it really makes the code. I do like consistency though.